google_nest_sdm/webrtc_util.py

```diff
@@ -0,0 +1,114 @@
+# SDP Direction Constants
+DIRECTION_SENDRECV = 'sendrecv'
+DIRECTION_SENDONLY = 'sendonly'
+DIRECTION_RECVONLY = 'recvonly'
+DIRECTION_INACTIVE = 'inactive'
+
+AVAILABLE_DIRECTIONS = [DIRECTION_SENDRECV,
+                        DIRECTION_SENDONLY,
+                        DIRECTION_RECVONLY,
+                        DIRECTION_INACTIVE]
+
+
+class WebRTCUtility:
+    """
+    This class provides methods for managing WebRTC SDP.
+    """
+
+    def get_media_direction(self, sdp, kind):
```

Comment:
Use mypy types rather than relying on type information in the arguments everywhere. (This fails pre-commit checks and the workflow checks)

Adding an enum for kind will make these self documenting then we don't need the argument documentation.

```
class Kind(StrEnum):
    AUDIO = 'audio'
    VIDEO = 'video'
```

```suggestion
    def get_media_direction(self, sdp: str, kind: Kind) -> str | None:
```